bazelrunner/src/main/java/org/jetbrains/bsp/bazel/bazelrunner/utils/Stopwatch.java

```diff
@@ -0,0 +1,23 @@
+package org.jetbrains.bsp.bazel.bazelrunner.utils;
+
+import java.time.Duration;
+
+public class Stopwatch {
```

Comment:
u can use guava stop watch as well if u want to: https://guava.dev/releases/24.0-jre/api/docs/com/google/common/base/Stopwatch.html